examples/petstore/lib/testdata/doc/openapi.golden.json

```diff
@@ -949,10 +949,5 @@
 			"description": "local server",
 			"url": "http://localhost:9999"
 		}
-	],
-	"tags": [
-		{
-			"name": "pets"
-		}
 	]
```

Comment:
I might add a for loop on `(s *Server) OutputOpenAPISpec()` that run on all the routes and generate all available tags.

Reply:
Option 1 is much easier to see and understand.

Reply:
This is done. This PR is now 100% not breaking!